File: Ecole--Edtech1/src/EnseignantBundle/Entity/Bulletin.php
<?php

namespace EnseignantBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Bulletin
 *
 * @ORM\Table(name="bulletin")
 * @ORM\Entity(repositoryClass="EnseignantBundle\Repository\BulletinRepository")
 */
class Bulletin
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var int
      @ORM\Column(name="eleve", type="integer")
     */
    private $eleve;

    /**
     * @var int
    @ORM\Column(name="trimestre", type="integer")
     */
    private $trimestre;

    /**
     * @return int
     */
    public function getTrimestre()
    {
        return $this->trimestre;
    }

    /**
     * @param int $trimestre
     */
    public function setTrimestre($trimestre)
    {
        $this->trimestre = $trimestre;
    }

    /**
     * @return int
     */
    public function getEleve()
    {
        return $this->eleve;
    }

    /**
     * @param int $eleve
     */
    public function setEleve($eleve)
    {
        $this->eleve = $eleve;
    }


    /**
     * @var string
      @ORM\Column(name="classe", type="string", length=255)
     */
    private $classe;

    /**
     * @return string
     */
    public function getClasse()
    {
        return $this->classe;
    }

    /**
     * @param string $classe
     */
    public function setClasse($classe)
    {
        $this->classe = $classe;
    }


    /**
     * @var float
     *
     * @ORM\Column(name="moyenne", type="float", nullable=true)
     */
    private $moyenne;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }


    /**
     * Set moyenne
     *
     * @param float $moyenne
     *
     * @return Bulletin
     */
    public function setMoyenne($moyenne)
    {
        $this->moyenne = $moyenne;

        return $this;
    }

    /**
     * Get moyenne
     *
     * @return float
     */
    public function getMoyenne()
    {
        return $this->moyenne;
    }
}

File: ProjetPi/src/com/mycompany/myapp/entities/Bulletin.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author dell
 */
public class Bulletin {
    private int id;
    private int eleve;
    private int trimestre;
    private int classe;
    private float moyenne;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getEleve() {
        return eleve;
    }

    public void setEleve(int eleve) {
        this.eleve = eleve;
    }

    public int getTrimestre() {
        return trimestre;
    }

    public void setTrimestre(int trimestre) {
        this.trimestre = trimestre;
    }

    public int getClasse() {
        return classe;
    }

    public void setClasse(int classe) {
        this.classe = classe;
    }

    public float getMoyenne() {
        return moyenne;
    }

    public void setMoyenne(float moyenne) {
        this.moyenne = moyenne;
    }

    @Override
    public String toString() {
        return "Bulletin{" + "id=" + id + ", eleve=" + eleve + ", trimestre=" + trimestre + ", classe=" + classe + ", moyenne=" + moyenne + '}';
    }

    public Bulletin() {
    }

    public Bulletin(int id, int eleve, int trimestre, int classe, float moyenne) {
        this.id = id;
        this.eleve = eleve;
        this.trimestre = trimestre;
        this.classe = classe;
        this.moyenne = moyenne;
    }

    public Bulletin(int eleve, int trimestre, int classe, float moyenne) {
        this.eleve = eleve;
        this.trimestre = trimestre;
        this.classe = classe;
        this.moyenne = moyenne;
    }
  
    
}
